CHANGELOG-1.2.md

```diff
@@ -22,6 +22,35 @@ Previous change logs can be found at [CHANGELOG-1.1](https://github.com/opencurv
     - [Support to list the volume on copyset where majority copy is failed.](https://github.com/opencurve/curve/pull/233)
 - [Clean up the unit test temporary folder.](https://github.com/opencurve/curve/pull/206/)
 
+
+Hardware: 6 nodes, each with:
```

Comment:
6 nodes as chunkserver

Don't need hardware configuration of client or mds ？

Reply:
> 6 nodes as chunkserver
> 
> Don't need hardware configuration of client or mds ？

Yes, the environmental information is from test report and consistent with the previous version